package/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "brave-ui",
   "main": "index.js",
   "sideEffects": false,
-  "version": "0.38.0",
+  "version": "0.34.4",
```

Comment:
We shouldn't need to make updates to this file anymore (unless we are doing an NPM publish)

Was this inadvertent or did `react-storybook-addon-chapters` need that bump?

Reply:
inadvertent for both

Reply:
Removed.